2/contrib/s2i/run

```diff
@@ -79,7 +87,6 @@ function install_plugins() {
   fi
   
   if [ "$(ls -A /opt/openshift/plugins 2>/dev/null)" ]; then
-    mkdir -p ${JENKINS_HOME}/plugins
     echo "Copying $(ls /opt/openshift/plugins | wc -l) Jenkins plugins to ${JENKINS_HOME} ..."
     cp -r /opt/openshift/plugins/* ${JENKINS_HOME}/plugins/
```

Comment:
doesn't this also need a --remove-destination?

Reply:
this is the vanilla first install path ... JENKINS_HOM/plugins is empty

Reply:
and in reality, this is centos specific now, and independent of all the rhel symlink history

Reply:
this function is also called when OVERRIDE_PV_PLUGINS_WITH_IMAGE_PLUGINS is set.

Reply:
so it would be called in both a rhel and centos image when that variable is set and install_plugins.sh put some plugin files in /opt/openshift/plugins

Reply:
so if they override an rpm installed plugin (which could have symlink) via this mechanism, they could need --remove-destination; otherwise, they cp will just overwrite

which we don't prevent, so sure (maybe we should prevent ... pretend I didn't say that)

this would be an existing bug then, which nobody has reported yet ... but sure, I'll switch to the other cp mechanism

Reply:
though the suffix should be jpi since install_plugins.sh is used, so slightly different

Reply:
yeah it's an existing bug.